### PR TITLE
Add OS Name into add-host-metadata

### LINKED ISSUE #8948: [Metricbeat]  Add Operating System Name to add_host_metadata processor
When using the `add_host_metadata` Metricbeat processor it would be useful to include the Operating System name so it returns the following field value in Windows

`Microsoft Windows Server 2012 R2 Datacenter`

Might also be worthy to add the same for other Operating Systems that Metricbeat supports so that you'd have fields similar to 

`Ubuntu 18.04 LTS`
`CentOS Linux 7`
`Red Hat Enterprise Linux Server 7.3`

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
This PR is for https://github.com/elastic/beats/issues/8948

closes #8948

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -92,6 +92,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha1...master[Check the HEAD d
 - The `elasticsearch/node` metricset now reports the Elasticsearch cluster UUID. {pull}8771[8771]
 - Add service.type field to Metricbeat. {pull}8965[8965]
 - Support GET requests in Jolokia module. {issue}8566[8566] {pull}9226[9226]
+- Add `os.name` field into libbeat add_host_metadat processor. {issue}8948[8948] {pull}9405[9405]
```
Comment (ruflin) [inline CHANGELOG.asciidoc]:
```suggestion
- Add `host.os.name` field to add_host_metadata processor. {issue}8948[8948] {pull}9405[9405]
```

Proposing this change as most users don't know what libbeat is and are mainly interested in the full path of the field.

File: CHANGELOG.asciidoc [hunk 2]
```diff
@@ -95,6 +95,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha1...master[Check the HEAD d
...
```
Comment (ruflin) [inline CHANGELOG.asciidoc]: @kaiyan-sheng The PR still seems to contain the old changelog?

Comment (kaiyan-sheng) [reply CHANGELOG.asciidoc]: oh man... sorry!! I somehow reversed it...

**Comment by exekias** [comment]
closes #8948

**Pull request merged**